pkg/action/loader.go

```diff
@@ -221,7 +221,9 @@ Action definition is correct, but dashes are not allowed in templates, replace "
 
 // ConvertInputToTplVars creates a map with input variables suitable for template engine.
 func ConvertInputToTplVars(input Input, ac *DefAction) map[string]interface{} {
-	values := make(map[string]interface{}, len(input.Args)+len(input.Opts))
+	predefinedVars := map[string]bool{"current_uid": true, "current_gid": true}
```

Comment:
Please, move the code out of the function, because it has 1 purpose - convert input to template data.
You may run the code after the invocation of the function `ConvertInputToTplVars`.

Reply:
changed, thx